helpers/prometheus.py

```diff
@@ -378,3 +379,21 @@ def collect(self):
                     )
 
             yield project_events_metrics
+
+        if self.rate_limit_metrics == "True":
+            project_rate_metrics = CounterMetricFamily(
```

Comment:
Using `GaugeMetricFamily` looks more appropriate here since this number can arbitrarily go up and down if the administrator changes the rate limit configuration, what do you think?

Reply:
I'm totally agree, Counter is for increasing values, Gauge is more appropriate :thumbsup: